services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableEntityManagerImpl.java

```diff
@@ -358,9 +365,23 @@ RowReferenceSet appendRowsToTable(UserInfo user, List<ColumnModel> columns,
 			refs.add(ref);
 		}
 		results.setRows(refs);
+		sendFileUploadEvents(user.getId(), delta);
```

Comment:
Exclude files that are already in the table: getFileHandleIdsAssociatedWithTable